frontend/src/services/UserService.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const BASE_URL = "http://localhost:8080";
+
+
+class UserService {
```

Comment:
Since it is stateless, no need to define a class. 
We can export the the functions only.